p2panda-discovery/README.md

```diff
@@ -1 +1,28 @@
-Awesome documentation goes here.
+<h1 align="center">p2panda-discovery</h1>
+
+<div align="center">
+  <img src="https://raw.githubusercontent.com/p2panda/.github/main/assets/panda-left.gif" width="auto" height="30px">
+  <strong>Your toolbox to build offline-first applications!</strong>
```

Comment:
I'd add the short description about the crate here instead of our "tagline", what do you think?

Reply:
Cool, that sounds good to me.